### PR TITLE
add vidulum

### PR DESCRIPTION
**Pull request by Corey-Code**
VIdulum VDL
https://vidulum.app

### CODE REVIEW

**Comment by IbrarMakaveli** [comment]
I think your formatter prettier of your editor make some correction on JSON File.
Can you undo this modification please, Thanks :)

**Comment by Corey-Code** [comment]
It's did and I will :)

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -859,6 +859,34 @@
...
+        }
+      ],
+      "base": "ibc/E7B35499CFBEB0FF5778127ABA4FB2C4B79A6B8D3D831D4379C4048C238796BD",
+      "name": "Vidulum",
+      "display": "vdl",
+      "symbol": "VDL",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-124",
+        "source_denom": "vdl"
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:885]: I don't think this is correct.  Should be `uvdl` right?

Comment (Corey-Code) [reply osmosis-1/osmosis-1.assetlist.json]: Yes just pushed fix

**Pull request merged**